BHoM_UI/Templates/MethodCaller.cs

```diff
@@ -133,58 +112,14 @@ public override object Run(object[] inputs)
             {
                 BH.Engine.Reflection.Compute.RecordError("The component is not linked to a method.");
                 return null;
-            }    
+            }
         }
 
 
         /*************************************/
         /**** Private Methods             ****/
         /*************************************/
 
-        protected virtual void CompileFunction()
```

Comment:
This has now been migrated to the Reflection_Engine as `Engine.Reflection.Convert.ToFunc`